examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1066,6 +1066,11 @@ def compute_text_embeddings(prompt, text_encoders, tokenizers):
         if args.train_text_encoder:
             text_encoder_one.train()
             text_encoder_two.train()
+            
+            # set top parameter requires_grad = True for gradient checkpointing works
```

Comment:
I don't fully understand why we add these changes. Can you explain a bit @shyammarjit ? How does this solve an error / warning with gradient checkpointing?

Reply:
@patrickvonplaten please have a look at this issue https://github.com/huggingface/diffusers/issues/5016 and its follow-up issue at https://github.com/kohya-ss/sd-scripts/issues/323.